CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing Guide
+
+Thank you for your interest in contributing to Enzyme! There are multiple ways to contribute, and we appreciate all contributions. In case you have questions, you can either use the [Discourse](https://discourse.llvm.org/c/projects-that-want-to-become-official-llvm-projects/enzyme/45) or the #enzyme channel on [Discord](https://discord.gg/xS7Z362).
+
+
+## Ways to Contribute
+
+### Bug Reports and Feature Requests
+
+If you are working with Enzyme, and run into a bug, or require additional features, we definitely want to know about it. Please let us know by either creating a [GitHub Issues](https://github.com/EnzymeAD/Enzyme/issues), or starting a thread in Enzyme's [LLVM Discourse Category](https://discourse.llvm.org/c/projects-that-want-to-become-official-llvm-projects/enzyme/45).
+
+### Bug Fixes
+
+If you are interested in contributing code to Enzyme, issues labeled with the [good-first-issue](https://github.com/EnzymeAD/Enzyme/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) keyword in the [GitHub Issues](https://github.com/EnzymeAD/Enzyme/issues) are a good way to get familiar with the code base. If you are interested in fixing a bug please comment on it to let people know you are working on it.
+
```

Comment:
I would actually recommend trying it out in the enzyme compiler explorer here.

Reply:
I just added a specific highlight to the compiler explorer.

Reply:
Oh I meant as a first way to test a bug, rather than compiling LLVM from source.

Reply:
How about now, i.e. Compiler Gym as the first default option, and only then resorting to a source build..